core/src/main/resources/static/js/browseAcls.js

```diff
@@ -258,43 +258,66 @@ app.controller("browseAclsCtrl", function($scope, $http, $location, $window) {
                     closeOnConfirm: true,
                     closeOnCancel: true
                 }).then(function(isConfirm){
-                    if (isConfirm.dismiss != "cancel") {
-                        $http({
-                                method: "POST",
-                                url: "createTopicDeleteRequest",
-                                headers : { 'Content-Type' : 'application/json' },
-                                params: {'topicName' : $scope.topicSelectedParam,
-                                                           'env' : env},
-                                data: {'topicName' : $scope.topicSelectedParam,
-                                                           'env' : env},
-                            }).success(function(output) {
-
-                                if(output.result == 'success'){
-                                    swal({
-                                         title: "",
-                                         text: "Topic Delete Request : "+output.result,
-                                         showConfirmButton: true
-                                     }).then(function(isConfirm){
-                                            $window.location.href = $window.location.origin + $scope.dashboardDetails.contextPath + "/myTopicRequests?reqsType=CREATED&deleteTopicCreated=true";
-                                        });
-                                }
-                                else{
-                                        $scope.alertTopicDelete = "Topic Delete Request : "+output.result;
-                                        $scope.alertnote = $scope.alertTopicDelete;
-                                        $scope.showSubmitFailed('','');
-                                    }
-                            }).error(
-                                function(error)
-                                {
-                                    $scope.handleValidationErrors(error);
-                                }
-                            );
+                    if (isConfirm.dismiss !== "cancel") {
+                        swal({
+                            title: "Delete schema ?",
+                            text: "Delete associated schema of the topic, if exists?",
+                            type: "warning",
```

Comment:
Maybe a slight change here
"Delete associated schema of the topic, if exists?"  ->
"Delete associated schema of this topic, if it exists?"

Reply:
Agree, done. Adding (all versions)